tests/js/client/shell/shell-restore-integration.js

```diff
@@ -1076,6 +1162,93 @@ function restoreIntegrationSuite () {
         db._useDatabase("_system");
       }
     },
+
+    testRestoreEnableRevisionTreesTrue: function () {
+      let path = fs.getTempFile();
+      try {
+        fs.makeDirectory(path);
+        let fn = fs.join(path, cn + ".structure.json");
+
+        fs.write(fn, JSON.stringify({
+          indexes: [],
+          parameters: {
+            name: cn,
+            type: 2
+          }
+        }));
+
+        let args = ['--collection', cn, '--import-data', 'false', '--enable-revision-trees', 'true'];
+        runRestore(path, args, 0); 
+
+        let c = db._collection(cn);
+        let props = c.properties();
+        assertTrue(props.hasOwnProperty("syncByRevision"));
+        assertTrue(props.syncByRevision);
```

Comment:
Why `usesRevisionsAsDocumentIds` is not checked? Or it is not added in this case?

Reply:
The attribute is not exposed in the properties in all versions. That's why the test checks `syncByRevision` only.